containers/Ideas/IdeasPage.styles.js

```diff
@@ -4,6 +4,12 @@ export const styles = StyleSheet.create({
   container: {
     flex: 1,
     backgroundColor: 'white',
-    padding: 32
+    padding: 32,
+    paddingBottom: 70
   },
+  absoluteBottomBtn: {
```

Comment:
Yeah, and I didn't really get why the button was already where it's meant to be without any layouting (apart from padding). I also didn't get it to be full width without breaking all the styling. :(